lib/bitters/generator.rb

```diff
@@ -76,5 +85,27 @@ def stylesheets_directory
     def top_level_directory
       File.dirname(File.dirname(File.dirname(__FILE__)))
     end
+
+    def command_exists?(command)
+      `which #{command}` != ""
+    end
+
+    def change_file_ext(file, ext)
+      no_ext = File.join(File.dirname(file), File.basename(file, ".*"))
+      "#{no_ext}.#{ext}"
+    end
+
+    def convert_syntax_to_sass
+      if command_exists?("sass-convert")
```

Comment:
Nevermind, I see the "command_exists?" method above. Not sure why it wasn't working for me, but I'll try again